src/qgitrepository.h

```diff
@@ -681,6 +667,31 @@ namespace LibQGit2
              */
             bool shouldIgnore(const QString &path) const;
 
+            /**
+             * Set the identity to be used for writing reflogs.
+             *
+             * @param name The name to use for the reflog entries
+             * @param email The email to use for the reflog entries
+             *
+             * @throws LibQGit2::Exception
+             */
+            void setIdent(const QString &name, const QString &email);
+
+            /**
+             * The identity used for writing reflogs.
```

Comment:
This type is just a name/email pair. It doesn't by itself determine for what it is used. Hence the documentation is incorrect. This type is so simple that IMO it can well be without any comment.

Reply:
Removed.